packages/hash/api/package.json

```diff
@@ -27,7 +27,7 @@
     "@snowplow/node-tracker": "3.3.1",
     "@types/lodash": "4.14.175",
     "@types/redis": "2.8.32",
-    "ajv": "8.7.1",
+    "ajv": "8.11.0",
```

Comment:
`block-scripts` came with a newer version, so I had to bump this one to avoid incompatible typings